### PR TITLE
tests/nested/manual/core20-remodel: try to work around midnight problem

### PR DESCRIPTION
**Pull request by anonymouse64**
Not sure how we can ensure that it will run right before midnight to trigger the bug, but I can try and set a timer to execute right before that time in my TZ to see if this works

### CODE REVIEW

File: tests/nested/manual/core20-remodel/task.yaml [hunk 1]
```diff
@@ -44,7 +41,16 @@ execute: |
...
     echo "Verify seed system with label $revno2_label"
-    tests.nested exec "sudo cat /run/mnt/ubuntu-seed/systems/${revno2_label}/model" > revno-2-from-seed.model
+    # this may fail if the recovery system was created just before midnight and
+    # then we capture label_base just after midnight, so try to work around this
+    # by trying yesterday's date if today's date doesn't work
+    if tests.nested exec "sudo test -d /run/mnt/ubuntu-seed/systems/${revno2_label}"; then
+        tests.nested exec "sudo cat /run/mnt/ubuntu-seed/systems/${revno2_label}/model" > revno-2-from-seed.model
+    else
+        echo "Checking alternate label from yesterday"
+        revno2_label=$(tests.nested exec "date --date=yesterday '+%Y%m%d'")
```
[the remark this answers is not in the document]
Comment (anonymouse64) [reply tests/nested/manual/core20-remodel/task.yaml:51]: yeah if the change description (or some other mechanism beyond just checking the /systems/ dir) has the recovery system label that would be the best to use I think

[the remark this answers is not in the document]
Comment (mardy) [reply tests/nested/manual/core20-remodel/task.yaml:51]: Another possibility is https://github.com/snapcore/snapd/commit/5d494e278ccfedc88ed7644a4d105cfb695357dd

[the remark this answers is not in the document]
Comment (anonymouse64) [reply tests/nested/manual/core20-remodel/task.yaml:51]: yes I think @mardy's solution is a bit better, when I get time I will go with that one instead

[the remark this answers is not in the document]
Comment (anonymouse64) [reply tests/nested/manual/core20-remodel/task.yaml:51]: Since I didn't get time to work on this, I'm going to close this PR

**Pull request closed without merging**